hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/interfaces/StorageLocationReportMXBean.java

```diff
@@ -33,6 +33,8 @@ public interface StorageLocationReportMXBean {
 
   long getRemaining();
```

Comment:
@vtutrinov @adoroszlai 
What is the definition of `remaining`, I noticed that `remaining` is actually Volume `available`. `available=capacity - reserved - used`.

- So does `committed` have to be reported to the SCM separately.
- Or is it added to `remaining`, which is defined as the amount of space on the Volume that is guaranteed to be available for data storage? `available=capacity - reserved - used - commited`.
We use `available` in a lot of places, and it seems like we should take `commited` into `available`.


https://github.com/apache/ozone/blob/9b3d045a9a2d87b67cc8523d166c3a82c4e05fa5/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/MutableVolumeSet.java#L473

Reply:
Yes, it looks like your initial solution is the more appropriate choice.